tests/test_modeling_common.py

```diff
@@ -3720,6 +3722,12 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
         for model_class in self.all_model_classes:
             config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
             model = model_class(config)
+            # FIXME: we deactivate boolean mask because pretrained models
+            # will not load the mask token
```

Comment:
Could you expand on this comment. Specifically, what needs to be fixed. 

I would expect that pretrained models would load the mask token, depending on whether `use_mask_token` defaults to True or False for the model itself.

Reply:
Just pushed a more comprehensive comment, let me know what you think :)